webapp-django/crashstats/crashstats/views.py

```diff
@@ -201,6 +201,39 @@ def get_timedelta_from_value_and_unit(value, unit):
     return date_delta
 
 
+def get_super_search_style_params(**kwargs):
+    if 'signature' not in kwargs:
+        raise ValueError('"signature" is a mandatory parameter')
+
+    params = {
+        'signature': kwargs.get('signature'),
```

Comment:
This should be `kwargs['signature']` since the use of `.get()` implies that it can not be there.